pkg/smbios/info64_test.go

```diff
@@ -116,33 +133,48 @@ func Test64Marshal(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error setup mock system info table: %v", err)
 	}
-	longTable.strings[0] += "a-very-long-string"
+	longStr := "a-very-loooooooooooooooooooooooooooooooooooong-string"
```

Comment:
do you think we could convert this test to one that doesn't use the mock data?

I think it'd be clearer what's being tested and how if we went from []*Table -> modify -> binary data, or binary data -> modify -> binary data in a tabled test.